infra-templates/pipeline/config.yml

```diff
@@ -0,0 +1,4 @@
+name: Rancher Pipeline 
+description: (Experimental) Rancher CI service
+version: 1.0.0
+category: Continuous Integration 
```

Comment:
Will this work with any orchestration or is it `cattle` specific? Let's add the following labels assuming it only works in cattle. 

```
labels:
  io.rancher.certified: Experimental
  io.rancher.orchestration.supported: 'cattle'
```

Reply:
updated